src/mudlet.qrc

```diff
@@ -178,7 +178,9 @@
         <file>mudlet-lua/lua/generic-mapper/generic_mapper.xml</file>
         <file>mudlet-lua/lua/utf8_filenames.lua</file>
         <file>mudlet-mapper.xml</file>
+        <file>Mudlet_splashscreen_development.png</file>
```

Comment:
It looks really rather "broken" - and from what I remember this was exactly the idea you're trying to convey - but it's not appealing to be using something that already looks broken from the get-go.

Reply:
Not broken, more a *work-in-progress* like the text hasn't been painted yet (but the can of paint is there waiting, resting on the hammer that will be needed  to put the last two letters up). The partially bordered up exit with the hazard tape is to reinforce that this version may not work properly and is not a finished release version.

Given that the other one is labelled with "PTB" would adding "DEV" in the same format help?

I had an alternative design in mind which I thought I prepared a long while back but cannot find which involved a really red sun just rising over the horizon (much lower than the moon in the PTB, and with a much stronger flare effect) and a strong red tint to the whole image (like something like a "vampire's eye" view in a paranormal fantasy film) - sort of indicating in a (not) subtle way that this version is working towards the *new* day of a Mudlet release - would that be preferable do you think?

As it is I do not think we should keep development versions (having something in APP_BUILD other than something beginning with `-public-test-build`) having the same splash screen as the release version.  Though we ought then remind packagers that if they follow our advice to add something to that variable to identify their version as a packaged one then either they will need to patch the splashscreen selection code in `main.cpp` and `dlgAboutDialog.cpp` OR we add ***another*** case that allows "branding"...

Reply:
> would that be preferable do you think?

I think that sounds good.